doc/man3/OSSL_CMP_SRV_CTX_new.pod

```diff
@@ -155,6 +158,11 @@ messages with POPO 'RAVerified'.
 OSSL_CMP_SRV_CTX_set_grant_implicit_confirm() enables granting implicit
 confirmation of newly enrolled certificates if requested.
 
+OSSL_CMP_SRV_CTX_centralKeygen_req() returns 1 if central key generation
+is requested i.e., the public key in the certificate request (I<crm> if non-NULL, 
+otherwise I<p10cr>) is NULL or has an empty key value (with length zero).
+Otherwise or on error it returns 0.
+
```

Comment:
```suggestion
OSSL_CMP_SRV_CTX_centralKeygen_req() returns 1 if central key generation
is requested i.e., the public key in the certificate request (I<crm> is taken if it is non-NULL, 
otherwise I<p10cr>) is NULL or has an empty key value (with length zero).
In case I<crm> is non-NULL, this is checked for consistency with its B<popo> field being NULL. 
Otherwise it returns 0, and on error a negative value.

```

Reply:
extended popo check statement as `In case I<crm> is non-NULL, this is checked for consistency with its B<popo> field (must be NULL if central key generation is requested else must be present).`